js/render/painter/use_program.js

```diff
@@ -8,56 +8,56 @@ var util = require('../../util/util');
 // readFileSync calls must be written out long-form for brfs.
 var definitions = {
```

Comment:
In a followup PR, I'll move this object to mapbox-gl-shaders and make it the main export.